docs/contributing/roles.md

```diff
@@ -0,0 +1,47 @@
+# Contributor roles
+
+Contributors are welcome and are greatly appreciated! Every little bit helps, and we give credit to them.
+
+This document aims to explain the current roles in the DAG Factory project.
+For more information, check [contributing](howto.md).
```

Comment:
This link is not functional yet, no? Should we remove this line for the time being?

Reply:
Great observation, @pankajkoti, just fixed in 7e0dcc13b611b690ffe8f5df38b010fd32e396df and [b643fa1](https://github.com/astronomer/dag-factory/pull/341/commits/b643fa190a7a83dcdae20c8b6bf663ca99a6f8eb)